fe/fe-core/src/main/java/com/starrocks/statistic/StatisticExecutor.java

```diff
@@ -308,7 +308,7 @@ private List<TResultBatch> executeDQL(ConnectContext context, String sql) {
         context.setQueryId(UUIDUtil.genUUID());
         Pair<List<TResultBatch>, Status> sqlResult = executor.executeStmtWithExecPlan(context, execPlan);
         if (!sqlResult.second.ok()) {
-            throw new SemanticException("Statistics query fail | Error Message [%s] | {} | SQL [%s]",
+            throw new SemanticException("Statistics query fail | Error Message [%s] | QueryId [%s] | SQL [%s]",
```

Comment:
May the SQL be too long to output in the Exception?

Reply:
Statistics collection is triggered automatically and will only be output to the log